server/database/migrations/20170104145003_adding_arcived_at_to_pull_request_review_requests.js

```diff
@@ -0,0 +1,10 @@
+exports.up = knex =>
+  knex.schema.table('pull_request_review_requests', table => {
+    table.timestamp('archived_at')
```

Comment:
does this also need to start off as null? will this automatically mark it as null in the archived_at column? 
.defaultTo(null)